octoprint_mrbeam/static/js/maintenance.js

```diff
@@ -346,6 +347,17 @@ $(function () {
             });
         };
 
+        self.notifyPrefilterMaintenanceRequired = function () {
+            new PNotify({
+                title: gettext("Filter capacity reached 70%"),
+                text: gettext(
+                "At this Level we highly recommend to have a visual check on the filter, to make sure the sensor is representing the fill level of your filter, as it heavily depends on what material you are processing with your device."
```

Comment:
We don't have a sensor here, this can be missleading. But if the design team said so we can keep it